src/createAnimatedComponent/InlinePropManager.ts

```diff
@@ -35,14 +45,20 @@ function inlinePropsHasChanged(
   return false;
 }
 
-function getInlinePropsUpdate(inlineProps: Record<string, any>) {
+function getInlinePropsUpdate(
+  inlineProps: Record<string, unknown> &
+    TransformsStyle &
+    Partial<TransformArrayItem>
```

Comment:
How can `inlineProps` be of type `Partial<TransformArrayItem>`?

Reply:
This function is recursive and it can be so only in second iteration, it happens when you use ` return getInlinePropsUpdate(transform);`